### PR TITLE
chore(Integration): Cleanup removed methods

### PR DESCRIPTION
**Pull request by GoldenAngel2**
**Please describe the changes this PR makes and why it should be merged:**
The method no longer exists (and/or bots don't have access to it)
> DiscordAPIError: 405: Method Not Allowed

**Status and versioning classification:**
Please move lines that apply to you out of the comment:
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR includes breaking changes (methods removed or renamed, parameters moved or removed)

### CODE REVIEW

**Comment by SpaceEEC** [comment]
The same applies to `Integration#edit` and `Guild#createIntegration`, doesn't it?
Neither are documented anymore. (Only get and delete seem to be https://discord.com/developers/docs/resources/guild#get-guild-integrations)

**Comment by GoldenAngel2** [comment]
Since it's not documented anymore, it's safe to say those should be removed aswell.

**Comment by GoldenAngel2** [comment]
`Guild#createIntegration`, `Integration#edit` and `Integration#sync` is now removed.

File: typings/index.d.ts [hunk 1]
```diff
@@ -3676,11 +3673,6 @@ export interface IntegrationData {
   type: string;
```
Comment (SpaceEEC) [inline typings/index.d.ts:3673]: This interface (`IntegrationData`) can now be deleted too.

**Review by SpaceEEC — CHANGES_REQUESTED** [review]

File: typings/index.d.ts (hunk 1 above)
Comment (GoldenAngel2) [reply typings/index.d.ts]: removed.

**Review by SpaceEEC — APPROVED** [review]
LGTM

**Review by iCrawl — APPROVED** [review]

**Pull request merged**